app/assets/stylesheets/templates/_hub-documents-index.scss

```diff
@@ -0,0 +1,13 @@
+.document-index-actions a {
+  color: inherit;
+}
+
+.index-table__cell:first-child {
```

Comment:
This isn't scoped to the documents-index as the filename suggest, so its going to bold the first child any time index-table__cell is applied in any table -- are we confident that's what we want?